apps/apple/FluentUITester/package.json

```diff
@@ -10,9 +10,9 @@
     "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos"
   },
   "dependencies": {
-    "react": "16.11.0",
-    "react-native": "0.60.0",
-    "react-native-macos": "^0.60.0-0"
+    "react": "16.9.0",
```

Comment:
Technically, this is moving down. Might as well bring this back up to 16.11.0 if we can?

Reply:
Maybe, consistency with what react-native generates is probably safer though

Reply:
Wil keep as is for now then